emrun.py

```diff
@@ -1653,6 +1654,9 @@ def run():
       logv('Web server root directory: ' + os.path.abspath('.'))
     else:
       logi('Web server root directory: ' + os.path.abspath('.'))
+    logv('Starting web server: http://%s:%i/' % (options.hostname, options.port))
+    httpd = HTTPWebServer((options.hostname, options.port), HTTPHandler)
+    options.port = httpd.socket.getsockname()[1]
```

Comment:
maybe add a comment here? otherwise it is a little odd to see port passed in and then read back.

Reply:
Good call. Added.

Reply:
Thanks!